### PR TITLE
feat: Cody page Implementation

### PR DESCRIPTION
**Pull request by gitstart-sourcegraph**
## Description
Implementation of the Cody landing page from the [design mockup](https://www.figma.com/file/oQgh5R0iSBG3UII6C8CPJ8/Cody-landing-page?node-id=20%3A1357&t=6q10YRBitZeMIgbD-0)

## Ref
[Sourcegraph issue](https://github.com/sourcegraph/about/issues/6055)
GitStart Ticket: [SGM-6055](https://developers.gitstart.com/client/sourcegraph/ticket/SGM-6055)

## Demo video
https://user-images.githubusercontent.com/51731962/224924894-79f57e8b-7283-4c50-98e5-3276e4417de0.mov

### CODE REVIEW

File: src/pages/cody.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+            </video>
+        </div>
+
+        <ContentSection
+            parentClassName="!py-0"
+            className="mx-auto flex flex-col items-center justify-center gap-x-8 py-[112px] md:flex-row"
+        >
+            <div className="max-w-[529px]">
+                <Heading size="h2" className="!text-4xl text-white">
+                    Cody is now in Private Beta
```
Comment (sqs) [inline src/pages/cody.tsx:80]:
```suggestion
                    Cody is now in private beta
```

File: src/pages/cody.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+                </Heading>
+                <p className="mt-6 text-lg text-gray-200">
+                    Cody works with Sourcegraph’s precise code intelligence and code search to provide even better
+                    results. Cody is now in Private Beta for select users.
```
Comment (sqs) [inline src/pages/cody.tsx:84]:
```suggestion
                    results.
```

File: src/pages/cody.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+                <Heading size="h5" className="text-white">
+                    Interested in deploying Cody for your team? Let us know.
+                </Heading>
+                <Link
+                    href="https://sourcegraph.typeform.com/to/pIXTgwrd"
+                    title="Private beta access request form"
+                    className="btn mt-8 bg-transparent pl-0 text-white"
+                    target="_blank"
+                >
+                    Request Private Beta access
```
Comment (sqs) [inline src/pages/cody.tsx:97]:
```suggestion
                    Join private beta waitlist
```

File: src/pages/cody.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+                    <ChevronRightIcon className="mb-1 !mb-0 ml-[10px] inline" />
+                </Link>
+            </div>
+        </ContentSection>
+
+        <CallToAction
+            title="Try the Sourcegraph app on your code"
```
Comment (sqs) [inline src/pages/cody.tsx:104]: What is the reason for having this CTA here? I'm afraid it will confuse users.

**Review by sqs — CHANGES_REQUESTED** [review]

**Review by iskyOS — APPROVED** [review]

**Pull request merged**